### DISCUSSION TITLE
How to remove unused media?

### DISCUSSION THREAD

**Discussion by br4yd** [anyproto/contributors#258] [Q&A] [answered]
Hi,
if I add a image to a note and then delete the note I can still see the image under Media objects. Is it possible to somehow delete unused media files that are not in any document? Because this fills up the storage.

**Comment by filip-anytype** [comment, marked as answer]
There's currently no easy way to clean up unused files, but we plan to improve this workflow soon as part of this project ([Clipboard and HTML Pasting enhancements](https://github.com/anyproto/roadmap/issues/79)).
